iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/concurrent/ThreadName.java

```diff
@@ -169,6 +169,7 @@ public enum ThreadName {
   UPGRADE_TASK("UpgradeThread"),
   REGION_MIGRATE("Region-Migrate-Pool"),
   STORAGE_ENGINE_RECOVER_TRIGGER("StorageEngine-RecoverTrigger"),
+  JVM_GC_STATISTICS_MONITOR("JVM-GC-Statistics-Monitor"),
```

Comment:
you can move this field after `JVM_PAUSE_MONITOR` and add it to `jvmThreadNames`